### PR TITLE
Feature/start page

### PR DESCRIPTION
**Pull request by ArtemLigerko**
WM-Screenshots-20221208201537

WM-Screenshots-20221208201549

WM-Screenshots-20221208201601

### CODE REVIEW

**Comment by ArtemLigerko** [comment]
The problem with `<wired-input>` is left, it's not working with useState, I'm didn't find solution and temporary used `<input>`

File: src/components/CreateGameForm.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+      <wired-dialog {...(isOpen ? { open: true } : {})}>
+        <Card>
+          <h1>Create game</h1>
+
+          {/* <p>
+            <wired-input type="text" pattern="/^\w+$/" onChange={handleInput} />
+          </p> */}
+
+          <p>
```
Comment (alexandrtovmach) [inline src/components/CreateGameForm.tsx]: not sure about using any wrapper here at all, but if you need it, then it shouldn't be a paragraph, maybe div?

File: src/components/CreateGameForm.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+const Card = styled.section`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  width: 400px;
+  height: 300px;
+
+  p {
```
Comment (alexandrtovmach) [inline src/components/CreateGameForm.tsx]: if you agreed to use styled components, then you should avoid using nested styles, and create separate component for each element

File: src/wired.d.ts [hunk 3]
```diff
@@ -1,26 +1,70 @@
 declare namespace JSX {
```
Comment (alexandrtovmach) [inline src/wired.d.ts]: please revert changes in this file, looks strange

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**